src/osm_configurator/model/project/configuration/aggregation_configuration.py

```diff
@@ -15,7 +19,13 @@ def __init__(self):
         """
         Creates a new instance of the AggregationConfiguration.
         """
-        pass
+        self._length = 2
```

Comment:
magic number which isnt documented nobody will know what thod even does

Reply:
this is from the config branch, it must be fixed there or it will create conflicts